hw/top_earlgrey/dv/cov/spi_host_device_unr_tied_off.el

```diff
@@ -0,0 +1,38 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: root
+// Format Version: 2
+// Date: Fri Nov 11 20:59:05 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "2715753694 4000582234"
```

Comment:
@eunchan 
@weicaiyang to double check, but this part looks good to me.

Reply:
This also LGTM